sway/tree/container.c

```diff
@@ -76,7 +77,7 @@ struct sway_container *container_create(enum sway_container_type type) {
 	return c;
 }
 
-static struct sway_container *_container_destroy(struct sway_container *cont) {
+static struct sway_container *container_finish(struct sway_container *cont) {
```

Comment:
`finish` is supposed to free all resources allocated by a struct without free-ing the struct itself. Might want to rename this function to something else.